lib/src/tag/pana_tags.dart

```diff
@@ -38,6 +38,7 @@ abstract class PanaTags {
   static const hasScreenshot = 'has:screenshot';
   static const isPlugin = 'is:plugin';
   static const isNullSafe = 'is:null-safe';
+  static const isSwiftPackageManagerReady = 'is:swiftpm-plugin';
```

Comment:
I think we should call the variable as close as the field value is, e.g. `isSwiftPMPlugin` or similar.

Reply:
Nice catch - forgot to rename this!